src/clean.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+function clean(ast, newObj, parent) {
+  ['range', 'loc'].forEach(name => {
+    delete newObj[name];
+  });
+}
+
+module.exports = clean;
```

Comment:
do you mind if I change this into:

```javascript
// eslint-disable-next-line no-unused-vars
const clean = (ast, newObj, parent) => {
  ['loc', 'range'].forEach(name => {
    if (newObj[name]) {
      delete newObj[name]; // eslint-disable-line no-param-reassign
    }
  });
};

module.exports = clean;
```

I am not a big fan of param reassign and unused vars, is it possible to change the signature of this function?

Reply:
You can delete the `parent` parameter since we don’t use it now, but I’m curious why you are checking if the key exists before deleting it. Shouldn’t it be harmless to delete something that doesn’t exist?

Reply:
yes my bad ... 😊 you are right no need to check; we can keep the parameter there if you think we may be need it later